packages/firestore/src/local/local_store.ts

```diff
@@ -658,6 +658,8 @@ export class LocalStore {
         return PersistencePromise.forEach(
           viewChanges,
           (viewChange: LocalViewChanges) => {
+            const targetId = viewChange.targetId;
```

Comment:
nit: You could use this instead of viewChange.targetId in the next 2 statements as well...

Reply:
Done